data/json/items/melee/misc.json

```diff
@@ -15,6 +15,24 @@
     "cutting": 3,
     "category": "weapons"
   },
+  {  
+    "id:": "bullwhip_razor",
+    "name": { "str: ""scourge" },
+    "type": "TOOL",
+    "description": "The \"cat 'o nine tails\", a handle with nine short leather whips each sporting a razor-sharp metal tip. This ancient instrument of torture causes massive bleeding but is an ineffecient weapon by design.",
+    "symbol": "/",
+    "color": "brown",
+    "weight": "3496",
+    "volume": 9,
+    "price": 4800,
+    "price_postapoc": 5200,
+    "to_hit": 2,
```

Comment:
base -2
grip: weapon +2
length: long +2
surface: every +1
balance: uneven -1
total is 2, you're good.